### ISSUE TITLE
Metal and Wooden Fences

### ISSUE THREAD

**Issue by Lukethero** [DayZMod/DayZ#313]
After an excruciating amount of looting and building of walls, I have come to an obvious conclusion. This conclusion is that metal walls are pointless. Metal walls require a shit ton of supplies, including screws ( which are horribly hard to find enough of). After testing from a preety reliable group ( the frogs) , satchel charges destroy both walls. All wooden walls need is some rough stone to start off with, some wood, and a few nails, which you find all over the place. After some personal teating, there is no billet penetration through either metal or wooden. Here is my question, why use metal walls? If there is some factor I am missing or wrong on, feel free to call me out on it.

**Comment by R4Z0R49** [comment]
If i remember correct im pretty sure we haven't released any change log saying the walls would have billet penetration if fact on the first release of the building system we did tell everyone they would take a good amount of time to be 100%. we simply do not have time to do everything in one go there's maybe 2 ppl still working on the mod. Most of everyone else moved onto A3 and other games.

For Now
Wooden walls added.
Build system updated to support snapping.
Gates added.
Break in system added.
Maintenance not added.

Metal walls was added by one of the other devs. They shouldn't really be in game yet as there no where close to how i see they should be. But as dayz is an alpha really no point pulling them all back out may as well leave them in and see how they progress.
